product_docs/docs/migration_portal/4/03_mp_using_portal/03_mp_quick_start.mdx

```diff
@@ -22,9 +22,9 @@ To migrate Oracle schemas using Migration Portal:
 
 1. Select the objects that aren't compatible with EDB Postgres Advanced Server.
 
-1. Refer to the Knowledge Base or interact with the [AI Copilot](/migration_portal/latest/03_mp_using_portal/mp_ai_copilot/) to look up and understand possible workarounds for the objects that aren't compatible in EDB Postgres Advanced.
+1.  To look up and understand possible workarounds for the objects that aren't compatible in EDB Postgres Advanced Server, refer to the Knowledge Base or interact with the [AI Copilot](/migration_portal/latest/03_mp_using_portal/mp_ai_copilot/).
    
-   !!! Important Important
```

Comment:
Using the Important admonition will put the word Important in. No need to enter it manually.